packages/mgt-components/src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -88,36 +101,40 @@
   .person-details-container {
     display: flex;
     flex-direction: column;
-    padding-inline-start: 23px;
-    padding-top: 23px;
+    padding-inline-start: 22px;
+    padding-block: 36px;
     border-bottom: 1px solid set-var(line-seperator__color__sub1, $theme-default, $common);
 
     .person-image {
-      --presence-background-color: #{set-var(presence-background-color, $theme-default, $person-theme)};
-      --presence-icon-color: #{set-var(presence-icon-color, $theme-default, $person-theme)};
-      --line1-font-size: var(--person-card-display-name-font-size, 20px);
-      --line1-line-height: var(--person-card-display-name-line-height, 28px);
-      --line1-color: #{set-var(line-1-color, $theme-default, $person-card-theme)};
-      --line2-font-size: var(--person-card-title-font-size, 14px);
-      --line2-line-height: var(--person-card-title-line-height, 20px);
-      --line2-color: #{set-var(line-2-color, $theme-default, $person-card-theme)};
-      --line3-font-size: var(--person-card-subtitle-font-size, 14px);
-      --line3-line-height: var(--person-card-subtitle-line-height, 19px);
-      --line3-color: #{set-var(line-3-color, $theme-default, $person-card-theme)};
-      --avatar-size: 75px;
-      --details-cta-spacing: 30px;
-      --user-presence-margin-offset: 5px;
+      --person-avatar-top-spacing: #{$person-card-avatar-top-spacing};
+      --person-details-left-spacing: #{$person-card-details-left-spacing};
+      --person-details-bottom-spacing: #{$person-card-details-bottom-spacing};
+      --person-background-color: #{$person-card-background-color};
+
+      --person-line1-font-size: #{$person-card-line1-font-size};
+      --person-line1-font-weight: #{$person-card-line1-font-weight};
+      --person-line1-text-line-height: #{$person-card-line1-line-height};
+      --person-line1-text-color: #{$person-card-line1-text-color};
+
+      --person-line2-font-size: #{$person-card-line2-font-size};
+      --person-line2-font-weight: #{$person-card-line2-font-weight};
+      --person-line2-text-line-height: #{$person-card-line2-line-height};
+      --person-line2-text-color: #{$person-card-line2-text-color};
+
+      --person-line3-font-size: #{$person-card-line3-font-size};
+      --person-line3-font-weight: #{$person-card-line3-font-weight};
+      --person-line3-text-line-height: #{$person-card-line3-line-height};
+      --person-line3-text-color: #{$person-card-line3-text-color};
+      --person-avatar-size-3-lines: #{$person-card-avatar-size};
+      --person-presence-wrapper-bottom: -15px;
```

Comment:
@gavinbarron how about a negative on the top property? I'm trying to fix a style issue on the avatar on person card, but the presence-wrapper is misplaced according to the design. The `top: -15` fixes this but I wonder if there's a better solution.

Reply:
Good question, there are times when we have to use negative margins to overlap elements, I think that this is one of those cases